test/bdd/features/verifiable_credential.feature

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+@all
+@verifiable
+Feature: Issue Verifiable Credential
+  @issue_vc_ldp
+  Scenario: Issue University Degree Credential with JWS Linked Data proof
+    Given "Stanford University" issues credential at "2018-03-15" regarding "Bachelor Degree" to "Alice"
+    When "Stanford University" adds "JWS Linked data" proof and issues the credential
```

Comment:
To me, looks like these 2 steps can be combined into one (both steps Issues Credential) - `"Stanford University" issues "Bachelor Degree" credential at "2018-03-15" to "Alice" with "JWS Linked data" proof`.

Reply:
this is true, you can add type at the end of first line e.g. with  "JWS Linked data" proof

Reply:
Agree, I just wanted to split this into two lines for me readability.

Reply:
It is more readable as Rolson suggested

Reply:
Corrected. The code now looks more concise and readable, thanks @rolsonquadras !